app/components/ChannelItem/index.jsx

```diff
@@ -19,8 +21,9 @@ export default class ChannelItem extends React.Component {
 
   constructor(props) {
     super(props);
+    const channelId = this.props.item.get('id');
     this.state = {
-      favorite: this.props.favorite || false,
+      favorite: !!this.props.local.get('favoritesChannels').find(id => id === channelId),
```

Comment:
`favouriteChannels`. But better to store it in the same way as `lastSeen` in my PR.